adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostBannerAd.m

```diff
@@ -55,6 +55,13 @@ - (void)getBannerWithSize:(GADAdSize)adSize {
   if (!strongConnector || !strongAdapter) {
     return;
   }
+    
+  if (SYSTEM_VERSION_LESS_THAN(kGADMAdapterMinimumOSVersion)) {
+    NSString* logError = [NSString stringWithFormat:@"Chartboost minimum supported OS version is iOS %@. Requested action is a no-op.", kGADMAdapterMinimumOSVersion];
```

Comment:
This line is definitely too long for Google. I think we should check out other examples and separate it in several lines following the style marked by existing code.

Reply:
Applies to other errors